README.md

```diff
@@ -888,66 +888,57 @@ end
 
 ## Hiera integration
 
-### Configuration
-
-Set the hiera config symbol properly in your spec files:
+At some point, you might want to make use of Hiera to bring in custom parameters for your class tests. In this section, we will
+provide you with basic guidance to setup Hiera implementation within rspec testing. For more information on Hiera, you should
+check our official [documentation](https://www.puppet.com/docs/puppet/8/hiera.html).
```

Comment:
should we link to latest instead? I'm not sure what happens with the link to 8 after the docs are removed:
```suggestion
check our official [documentation](https://www.puppet.com/docs/puppet/latest/hiera.html).
```

Reply:
yeah, its probably better practice to link to latest, although the link to 8 should be valid for another couple of years. At the very least, until Puppet 8 is deprecated, which would happen with the arrival of Puppet 10